doc/whats_new/v1.4.rst

```diff
@@ -168,6 +168,9 @@ Changelog
   for CSR × CSR,  Dense × CSR, and CSR × Dense datasets is now 1.5x faster.
   :pr:`26765` by :user:`Meekail Zain <micky774>`
 
+- |API| adding :func:`metric.paired_haversine_distances` to compute pairwise haversine distances.
+  :pr:`22705` by :user:`Jona Sassenhagen <jona-sassenhagen>` and :user:`Charles Patel <acharles7>`.
+
```

Comment:
The entry will need to be changed to focus on `paired_distancecs(..., metric="haversine")`

Reply:
👍